packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -400,7 +401,7 @@ $expand-less-icon: map-merge(
         color: color("white");
         line-height: line-height($theme-navigation-font-family, 3);
         display: block;
-        padding: units(1);
+        @include u-padding-x(2);
```

Comment:
We like to keep our `@include`'s at the top of the style definitions. Could you just move this to line `401` above the color definition?

Reply:
Done.